server/src/firebase.ts

```diff
@@ -1,5 +1,39 @@
-import { initializeApp } from 'firebase-admin/app'
+import * as admin from 'firebase-admin/app'
 import { getFirestore } from 'firebase-admin/firestore'
 
-const firebaseApp = initializeApp()
+// Service Account info needed in emulator
+const serviceAccount = {
+    // local service account
+}
+
+// To fix type errors
+const key = serviceAccount as admin.ServiceAccount
+
+let firebaseApp
+
+const use_emulator = false
```

Comment:
You can get the version of the node environment - I think this is process.env.NODE_ENV === 'production' to check